contracts/governance/airdrop.cairo

```diff
@@ -0,0 +1,2463 @@
+%lang starknet
```

Comment:
also, the interface file was not updated

Reply:
there's no interface file for the governance contract, ABI is generated automatically... If you mean ABI (for David), added in a new commit now